### DISCUSSION TITLE
Purpose | iDempiere

### DISCUSSION THREAD

**Discussion by giscus** [idempiere/idempiere.github.io#2] [Announcements]
# Purpose | iDempiere

The purpose of this site is to help you learn, configure, enhance, deploy, scale and support an iDempiere instance and the iDempiere community as a whole.

https://idempiere.github.io/docs/introduction

**Comment by cboecking** [comment]
Recommend the following primary structure:
* basic installation
* basic functional
* basic development
* manual

**Reply by frankmannheim** [reply]
This looks good. 
I wonder whether I must be logged in explicitly. (Frank) ... Ok I was logged in from things some days ago.

**Reply by cboecking** [reply]
I could not make a comment until I was a) logged in and b) gave giscus authorization to access my account.
